examples/offscreen-canvas.worker.js

```diff
@@ -128,6 +127,20 @@ const maxTotalLoading = 8;
 const maxNewLoads = 2;
 
 worker.addEventListener('message', (event) => {
+  if (event.data.action === 'requestFeatures') {
+    layers.forEach((layer) => {
+      if (inView(layer.getLayerState(), frameState.viewState)) {
+        layer.getFeatures(event.data.pixel).then((res) => {
+          worker.postMessage({
+            action: 'getFeatures',
+            features: JSON.parse(stringify(res)),
```

Comment:
When the features are converted to and from json the class information is lost it would make sense to call getProperties() here and only pass that information, otherwise you have to rely on private properties to get any information out of that object.